lua/nvim-tree/actions/tree/modifiers/expand-all.lua

```diff
@@ -64,9 +79,10 @@ local function gen_iterator()
 end
 
 ---@param base_node table
-function M.fn(base_node)
+function M.fn(base_node, expand_until)
```

Comment:
Please use an options table similar to `api.tree.resize`:
* add a class to `api.lua` to describe this
* docuement this at `*nvim-tree-api.tree.expand_all()*`